samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -385,6 +390,24 @@ static int fast_pair_prepare(void)
 		return err;
 	}
 
+	err = app_fp_adv_trigger_register(fp_adv_trigger_ak_write);
+	if ((err < 0) && (err != -EALREADY)) {
```

Comment:
Maybe we could assert that there would be no error? Error should not happen here unless the configuration was misaligned, so error check could be simplified to assert